ldm/dream/server.py

```diff
@@ -133,41 +148,46 @@ def image_progress(sample, step):
                 {'event':'step', 'step':step, 'url': url}
             ) + '\n',"utf-8"))
 
-        if initimg is None:
-            # Run txt2img
-            self.model.prompt2image(prompt,
-                                    iterations=iterations,
-                                    cfg_scale = cfgscale,
-                                    width = width,
-                                    height = height,
-                                    seed = seed,
-                                    steps = steps,
-                                    gfpgan_strength = gfpgan_strength,
-                                    upscale         = upscale,
-                                    sampler_name    = sampler_name,
-                                    step_callback=image_progress,
-                                    image_callback=image_done)
-        else:
-            # Decode initimg as base64 to temp file
-            with open("./img2img-tmp.png", "wb") as f:
-                initimg = initimg.split(",")[1] # Ignore mime type
-                f.write(base64.b64decode(initimg))
-
-            # Run img2img
-            self.model.prompt2image(prompt,
-                                    init_img = "./img2img-tmp.png",
-                                    iterations = iterations,
-                                    cfg_scale = cfgscale,
-                                    seed = seed,
-                                    steps = steps,
-                                    sampler_name    = sampler_name,
-                                    gfpgan_strength=gfpgan_strength,
-                                    upscale         = upscale,
-                                    step_callback=image_progress,
-                                    image_callback=image_done)
-
-            # Remove the temp file
-            os.remove("./img2img-tmp.png")
+        try:
+            if initimg is None:
```

Comment:
This change is mostly whitespace (I added a couple `try`s), so I recommend viewing the diff [with whitespace suppressed](https://stackoverflow.com/questions/37007300/how-to-ignore-whitespace-in-github-when-comparing/51755490#51755490).